tests/tls/tls-ja3s/test.yaml

```diff
@@ -6,6 +6,7 @@ requires:
     - HAVE_LIBJANSSON
     - HAVE_NSS
     - HAVE_LUA
+    - HAVE_JA3
```

Comment:
Could be its own prerequisite commit

Reply:
I think I'll separate all JA3 related changes out into a separate commit; might be cleaner.